<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TCapture</class>
 <widget class="QWidget" name="Gui::Pref::TCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="capture_tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="capture_tab">
      <attribute name="title">
       <string>Capture</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>12</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="icon_label">
           <property name="text">
            <string notr="true">TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&amp;Directory to store screenshots and captured video:</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="buddy">
              <cstring>screenshot_edit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="FileChooser" name="screenshot_edit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="screenshots_group">
         <property name="title">
          <string>&amp;Enable screenshots</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="screenshots_verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="screenshot_template_label">
              <property name="text">
               <string>Temp&amp;late:</string>
              </property>
              <property name="buddy">
               <cstring>screenshot_template_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="screenshot_template_edit"/>
            </item>
            <item>
             <widget class="QLabel" name="screenshot_format_label">
              <property name="text">
               <string>Fo&amp;rmat:</string>
              </property>
              <property name="buddy">
               <cstring>screenshot_format_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="screenshot_format_combo"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="subtitles_on_screeshots_check">
            <property name="text">
             <string>&amp;Include subtitles on screenshots</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileChooser</class>
   <extends>QLineEdit</extends>
   <header>filechooser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>capture_tab_widget</tabstop>
  <tabstop>screenshot_edit</tabstop>
  <tabstop>screenshots_group</tabstop>
  <tabstop>screenshot_template_edit</tabstop>
  <tabstop>screenshot_format_combo</tabstop>
  <tabstop>subtitles_on_screeshots_check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
